pkg/manifests/config.go

```diff
@@ -467,6 +476,35 @@ func (c *Config) NoProxy() string {
 	return c.ClusterMonitoringConfiguration.HTTPConfig.NoProxy
 }
 
+func (c *Config) LoadEnforcedBodySizeLimit(pcr PodsCapacityReader, ctx context.Context) error {
+	enforcedBodySizeLimit := c.ClusterMonitoringConfiguration.PrometheusK8sConfig.EnforcedBodySizeLimit
+	if enforcedBodySizeLimit == "" {
```

Comment:
From my POV, it is more readable to return early in case the value isn't empty. Then the function can focus on computing the correct value.

```suggestion
	if c.ClusterMonitoringConfiguration.PrometheusK8sConfig.EnforcedBodySizeLimit != "" {
		return pm.ValidateSizeField(c.ClusterMonitoringConfiguration.PrometheusK8sConfig.EnforcedBodySizeLimit)
	}
```

Reply:
yup, it is more readable in this way :)